package/libs/openssl/Makefile

```diff
@@ -100,18 +135,74 @@ $(call Package/openssl/Default/description)
 This package contains the OpenSSL command-line utility.
 endef
 
+define Package/libopenssl-gost
+  $(call Package/openssl/Default)
+  SUBMENU:=SSL
+  TITLE:=Russian GOST algorithms engine
+  DEPENDS:=libopenssl @OPENSSL_ENGINE @OPENSSL_USE_VERSION_1_0_2 +@OPENSSL_WITH_GOST
```

Comment:
Such selecting dependencies will cause features to always get enabled in the binary package repos, increasing the size of the main library. Is `GHOST` intended to be enabled by default? If so, you cna leave as-is.

Reply:
Ah, I missed the fact that it only applies to 1.0.2. Please disregard my comment then if it does not apply to 1.1.1 anyway.